packages/core/src/lib/pattern_engines.js

```diff
@@ -86,52 +99,106 @@ const PatternEngines = Object.create({
   loadAllEngines: function (patternLabConfig) {
     const self = this;
 
-    // Try to load engines! We scan for engines at each path specified above. This
-    // function is kind of a big deal.
-    enginesDirectories.forEach(function (engineDirectory) {
-      const enginesInThisDir = findEngineModulesInDirectory(
-        engineDirectory.path
-      );
+    // Try to load engines! We load the engines configured in patternlab-config.json
+    const enginesInConfig = findEnginesInConfig(patternLabConfig);
 
-      logger.debug(
-        `Loading engines from ${engineDirectory.displayName}: ${engineDirectory.path} ...`
-      );
+    if (enginesInConfig) {
+      // Quick fix until we've removed @pattern-lab/engine-mustache, starting with https://github.com/pattern-lab/patternlab-node/issues/1239
+      // @TODO: Remove after removing @pattern-lab/engine-mustache dependency
+      enginesInConfig.mustache = enginesInConfig.mustache || {};
```

Comment:
Interesting approach. Do we need to change this to handlebars after you changed the package with #1456?

Reply:
It's mainly a mitigation, because we still need the mustache engine as we reference mustache templates all over the place.